src/index.js

```diff
@@ -119,8 +134,12 @@ function getRootDir(options) {
 }
 
 function getIncludeDirs(options) {
+  const includes = []
+    .concat(options.include || [])
```

Comment:
Why the concat here?  You don't mutate `options.include` anywhere later in the function, and as you call map straight after you any accidental mutation won't affect the result here anyways

Reply:
Because include also accepts a string, and this creates an array from either a string or array of strings.

Reply:
aaaaaaaaah